css/my_portfolio.css

```diff
@@ -352,3 +351,83 @@ button:active {
 .margin-05 {
   margin: 0 5%;
 }
+
+#contact-us {
+  height: 100vh;
+  padding-top: 30%;
+  text-align: center;
+}
+
+.contact-us-text {
+  width: 100%;
+  text-align: center;
+  padding: 0 7%;
+}
+
+.contact-us-paragraph {
+  text-align: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 27px;
+  line-height: 40px;
+}
+
+.form {
+  text-align: center;
+  padding: 8%;
+}
+
+.form-control {
+  margin-bottom: 10px;
+  padding-bottom: 20px;
+}
+
+.form-control input {
+  border: 0 solid #979493;
+  display: block;
+  padding: 10px;
+  width: 100%;
+  background: #e5e5e5;
+  font-family: 'Inter', sans-serif;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.form-control textarea {
+  padding: 10px;
+  border: 0 solid #979493;
+  background: #fbf8f7;
+  color: #3c3a39;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+}
```

Comment:
- [ ] I think it will be nice to use `width: 100%` for the `textarea` after styling the form area with **_flexbox_**. This would make the `textarea` form control align with the design in the _Figma_ template.